src/components/molecules/HomeImageContainer.vue

```diff
@@ -29,7 +31,10 @@
 		width: 100vw;
 		transition: left 0.4s;
 	}	
-	.hidden {
+	.home__img-container--hidden {
```

Comment:
When you will start implementing sass please take attention on how it should be structured
For reference:
https://medium.com/@andrew_barnes/bem-and-sass-a-perfect-match-5e48d9bc3894